start.sh

```diff
@@ -53,7 +58,10 @@ case $CLUSTER_MODE in
            COMPOSE_FILE_COMMAND+=("-f" "docker-compose-federated.yml");;
 esac
 
-"${COMPOSE_FILE_COMMAND[@]}" down
```

Comment:
Is this the right thing to do, given we've got a separate stop script?
Successive runs of the start will replace containers only if they've been changed. In two minds.
Pro: Nicely separates start from stop
Con: Replacing the stack is 2 steps, not 1

Reply:
I'm not sure if I understand what you're saying. Starting things twice (without a 'stop' in between) is guaranteed to be problematic, becaue of port collisions. An implicit "first bring the old system down" which I think this is, might be sensible. In either case, not a hill for me to die on. We can test how one approach works, and change it later if we find we prefer another approach.

Reply:
Starting twice won't be a problem. Docker takes of that, and updates the existing stack. The "down" does more than stop the containers - it'll remove data too. I think you're right though - a clean start feels better. I'll revert that line.